package.json

```diff
@@ -29,6 +30,12 @@
     "enzyme": "^3.6.0",
     "enzyme-adapter-react-16": "^1.5.0",
     "enzyme-to-json": "^3.3.4",
+    "eslint": "^5.6.0",
```

Comment:
Once we've adopted `eslint` for linting, please kindly create a `script` to allow others run your lint, not just by editor plugins. e.g.

```json
"scripts": {
  ...
  "lint": "./node_modules/.bin/eslint src",
  "lint:fix": "./node_modules/.bin/eslint src --fix"
},
```


ref: https://eslint.org/docs/user-guide/command-line-interface#options

Reply:
By doing so, it allowed to lint your code not only `local` but also `CI` environment.
Please feel free to enhance your automatic test workflow.